embark.el

```diff
@@ -2990,8 +2990,8 @@ and leaves the point to the left of it."
   ("c" customize-face)
   ("b" make-face-bold)
   ("B" make-face-unbold)
-  ("i" make-face-italic)
-  ("I" make-face-unitalic)
+  ("/" make-face-italic)
+  ("|" make-face-unitalic)
   ("!" invert-face)
   ("gf" set-face-foreground)
```

Comment:
If we do this, how about `*` for bold? I can't think of a good binding for unbold, though. I'd like to avoid the prefix `g` for foreground and background.

Reply:
What about +- for bold/ubold? Do you have a proposal for foreground/background?

Reply:
Yes, `f` for foreground and `b` for background. Which is why I'm trying to get rid of `b` for bold. I like your `+`, `-` suggestion.

Reply:
+- for bold unbold and f/b for foreground/background.